rules/Php55/Rector/String_/StringClassNameToClassConstantRector.php

```diff
@@ -179,6 +179,10 @@ private function shouldSkip(string $classLikeName): bool
             if ($this->nodeNameResolver->isStringName($classLikeName, $classToSkip)) {
                 return true;
             }
+
+            if (fnmatch($classToSkip, $classLikeName, FNM_NOESCAPE)) {
```

Comment:
Maybe we can throw when wildcards are contained to assis the user in the upgrade path?

Reply:
Here it's actually expected, as both formats are supported. 

But we could inform in the `isName()` method to inform about BC break :+1:

Reply:
Thats what I had in mind, right :-)